content/news/2022/01/31/release-statefun-3.2.0.html

```diff
@@ -33,6 +33,8 @@
       /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
       /* We explicitly disable cookie tracking to avoid privacy issues */
       _paq.push(['disableCookies']);
+      /* Measure a visit to flink.apache.org and nightlies.apache.org/flink as the same visit */
+      _paq.push(["setDomains", ["*.flink.apache.org","*.nightlies.apache.org/flink"]]);
```

Comment:
I was not expecting this to appear in this PR, but it seems that you've found one page where this wasn't properly set yet when you were rebuilding the website. Thanks :)

Reply:
It's my pleasure